types/sdk.js

```diff
@@ -60,7 +60,7 @@ export type AuthMiddlewareOptions = {
   scopes: Array<string>;
 }
 export type HttpMiddlewareOptions = {
-  host?: string;
+  host: string;
```

Comment:
They are not used to generate the doc, but it's good for knowing the object shape when coding or whatever. The default value that we provide should not affect the type definition. If there is no other way I would prefer to do it like I did, so that the type definition is correct.